.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,12 @@ https://matplotlib.org/devdocs/devel/index.html-->
 example "Raises ValueError on Non-Numeric Input to set_xlim".  Please avoid
 non-descriptive titles such as "Addresses issue #8576".-->
 
+<!--If you are contributing fixes to docstrings
+(which are very welcome!), please pay attention to
```

Comment:
Can we make this less passive aggressive?

Reply:
Feel free to propose an alternate wording 😃

Reply:
I'm worried that the template is getting too large for anyone to bother reading - I'm happy to just get rid of (which are very welcome!) here,